test/NHSD.BuyingCatalogue.Documents.API.Integration.Tests/Steps/AzureBlobSteps.cs

```diff
@@ -6,9 +6,10 @@
 namespace NHSD.BuyingCatalogue.Documents.API.IntegrationTests.Steps
 {
     [Binding]
-    class AzureBlobSteps
+    internal class AzureBlobSteps
     {
         private AzureBlobStorageScenarioContext context;
+
         public AzureBlobSteps(AzureBlobStorageScenarioContext context)
         {
             this.context = context;
```

Comment:
'this' can be omitted if adding an underscore to the `context` field.

Reply:
👍